.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: node_js
+node_js:
+  - 0.10
```

Comment:
Also adding it to `find-open-port`: https://github.com/testiumjs/find-open-port/pull/1